### PR TITLE
docs: Fix broken links to user guide

### PR DESCRIPTION
**Pull request by lukemanley**

### CODE REVIEW

File: docs/source/user-guide/plugins/index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Plugins
+
+In this chapter you will learn how to write plugins in order to extend the Polars api with
+user-defined functions.
```
Comment (rodrigogiraoserrao) [inline docs/source/user-guide/plugins/index.md:4]:
```suggestion
In this chapter you will learn how to write plugins in order to extend the Polars API.
```

**Review by rodrigogiraoserrao — CHANGES_REQUESTED** [review]
Thanks a lot for picking these up!
Only one tiny change requested :D

File: docs/source/user-guide/plugins/index.md (hunk 1 above)
Comment (lukemanley) [reply docs/source/user-guide/plugins/index.md]: updated

**Review by rodrigogiraoserrao — APPROVED** [review]
Thanks for these!

**Pull request merged**